cmd/ci-operator-checkconfig/main.go

```diff
@@ -51,6 +54,9 @@ func (o *options) parse() error {
 	if err := o.loadResolver(registryDir); err != nil {
 		return fmt.Errorf("failed to load registry: %w", err)
 	}
+	if err := o.createCIOpConfigAgent(); err != nil {
+		return fmt.Errorf("failed to create CI Op config agent: %w", err)
+	}
```

Comment:
We could just inline that function, it's more trouble to jump to its definition than to read this:

```suggestion
	if agent, err := agents.NewConfigAgent(o.ConfigDir); err != nil {
		return fmt.Errorf("failed to create CI Op config agent: %w", err)
	} else {
		o.ciOPConfigAgent = agent
	}
```